<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>677</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>.AppleSystemUIFont</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="allLineButton">
              <property name="text">
               <string>总地铁线路</string>
              </property>
              <property name="default">
               <bool>true</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="transferButton">
              <property name="text">
               <string>换乘结果查询</string>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="labelTransfer">
              <property name="minimumSize">
               <size>
                <width>280</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>PingFang SC</family>
                <pointsize>16</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>起始点选择：</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout" columnstretch="1,4,4" columnminimumwidth="1,4,4">
              <item row="0" column="0">
               <widget class="QLabel" name="startLabel">
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>PingFang SC</family>
                 </font>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;起点：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="buddy">
                 <cstring>startLineComboBox</cstring>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="startLineComboBox">
                <property name="minimumSize">
                 <size>
                  <width>90</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>PingFang SC</family>
                 </font>
                </property>
                <item>
                 <property name="text">
                  <string>选择路线</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QComboBox" name="startStationComboBox">
                <property name="minimumSize">
                 <size>
                  <width>90</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>PingFang SC</family>
                 </font>
                </property>
                <item>
                 <property name="text">
                  <string>选择站点</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="destinationLabel">
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>PingFang SC</family>
                 </font>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;终点：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="buddy">
                 <cstring>dstLineComboBox</cstring>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="dstLineComboBox">
                <property name="minimumSize">
                 <size>
                  <width>90</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>PingFang SC</family>
                 </font>
                </property>
                <item>
                 <property name="text">
                  <string>选择路线</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QComboBox" name="dstStationComboBox">
                <property name="minimumSize">
                 <size>
                  <width>90</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>PingFang SC</family>
                 </font>
                </property>
                <item>
                 <property name="text">
                  <string>选择站点</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="minimumSize">
               <size>
                <width>190</width>
                <height>60</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>PingFang SC</family>
                <pointsize>16</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="title">
               <string>乘车策略：</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QRadioButton" name="minTransferButton">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>15</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>PingFang SC</family>
                   <pointsize>14</pointsize>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>最少换乘次数</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="minTimeButton">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>15</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>PingFang SC</family>
                   <pointsize>14</pointsize>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>最短所需时间</string>
                 </property>
                 <property name="checked">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGraphicsView" name="graphicsView">
          <property name="font">
           <font>
            <family>PingFang SC</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="labelTransferRoute">
          <property name="minimumSize">
           <size>
            <width>280</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>PingFang SC</family>
            <pointsize>16</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>查询路线结果：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="routeTextBrowser">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>PingFang SC</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
